src/main/java/com/google/codeu/data/FoodItem.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.codeu.data;
+
+
+/** A single message posted by a user. */
+public class FoodItem {
+
+  private String name;
+  public String amount;
+  public double co2peryear;
+
+  public FoodItem(String name, String amount, String country, double co2peryear) {
```

Comment:
If we’re not doing anything with country in this particular file, I don’t think it makes sense to carry it in the constructor. The country bit could come in the data analysis but with the charts because we’re getting that information from the user profile anyways